packages-next/auth/src/types.ts

```diff
@@ -48,6 +48,8 @@ export type AuthConfig<GeneratedListTypes extends BaseGeneratedListTypes> = {
   passwordResetLink?: AuthTokenTypeConfig;
   /** "Magic link" functionality */
   magicAuthLink?: AuthTokenTypeConfig;
+  /** Session data population */
+  sessionData?: { query: string };
```

Comment:
This seems like weird / unnecessary nesting.

I know it's clear, because this is the `query` that populates `session.data` and it has all those words in it, and I see the similarity to the items API, but I also can't imagine anything else that might go in this object... so why nest it?

I think I'd prefer just

```suggestion
  sessionData?: string;
```